packages/atauth/src/enroll_namespace.h

```diff
@@ -0,0 +1,32 @@
+#ifndef ATAUTH_ENROLL_NAMESPACE_H
+#define ATAUTH_ENROLL_NAMESPACE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdlib.h>
+
+struct enroll_namespace {
+  size_t len;
+  char **namespaces; // array of strings
+  enum namespace_permissions *permissions;
+};
+
+// read bit 1
+// write bit 2
+enum namespace_permissions {
+  np_readonly = 0b01,
+  np_read_write = 0b11,
```

Comment:
using a bit mask for permissions, so that when we parse 'rw' and 'wr', they both result in 0b11.